### ISSUE TITLE
Telemetry for Assets Managed by Blackbox

### ISSUE THREAD

**Issue by mikeghen** [crypdex/blackbox#8]
I committed 0.1 BTC to Sparkswap's exchange using blackbox. I was wondering if we could add some telemetry so I could opt into reporting that back to Crypdex.

**Comment by dmichael** [comment]
Would the context of this be to track a sort of "collective AUM"?

**Comment by mikeghen** [comment]
Yes, and I think just general usage would be wise to report back, let people opt in of course. Maybe even adding universal logging and just having a environment variable that could be toggle to forward logs to a crypdex log server, that would solve the global usage reporting issue, could be something simple and serverless, again, something to opt into
